remoulade/api/main.py

```diff
@@ -0,0 +1,42 @@
+""" This file describe the API to get the state of messages """
+from flask import Flask, jsonify, make_response, request
+from werkzeug.exceptions import BadRequest, NotFound
+
+import remoulade
+from remoulade.state import StateNamesEnum
+
+app = Flask(__name__)
+
+
+@app.route("/messages/states/")
+def get_states():
+    name_state = request.args.get("name")
```

Comment:
I tried to make a simpler implementation of this method: 
```
name_state = request.args.get("name")
    backend = remoulade.get_broker().get_state_backend()
    data = [s.asdict() for s in backend.get_states()]
    if name_state is not None:
        if name_state.lower() not in [state_enum.name.lower() for state_enum in StateNamesEnum]:
            raise BadRequest("{} state is not defined.".format(name_state))
        data = [s for s in data if s["name"] == name_state.lower()]
    return {'result': data}
```
I try to call only once `get_states` to keep things clear.
And it's more pythonic to `if a not in [1, 2]` as it's exaclty what you want to test, if a value is in a array